src/java.base/share/classes/jdk/internal/loader/NativeLibraries.java

```diff
@@ -396,6 +403,20 @@ boolean open() {
         }
     }
 
+    public static final NativeLibrary defaultLibrary = new NativeLibraryImpl(Object.class, "<default>", true, true) {
+
+        @Override
+        boolean open() {
+            throw new UnsupportedOperationException("Cannot load default library");
+        }
+
+        @Override
+        public long find(String name) {
+            return NativeLibraries.findEntryInProcess(name);
```

Comment:
is this AIX specific, it seems like the native is only defined in AIX

Reply:
Yes, this is only specific to AIX.  So I just updated the code to add an OS check when defining `defaultLibrary` given it is only valid on AIX.